test/commands/templates/install.test.js

```diff
@@ -14,6 +14,32 @@ const TheCommand = require('../../../src/commands/templates/install')
 const BaseCommand = require('../../../src/BaseCommand')
 const { TEMPLATE_PACKAGE_JSON_KEY, readPackageJson, writeObjectToPackageJson, getNpmDependency } = require('../../../src/lib/npm-helper')
 
+// mock project-installation calls
+jest.mock('@adobe/aio-lib-console-project-installation')
+const TemplateHandler = require('@adobe/aio-lib-console-project-installation')
+const mockTemplateHandlerInstance = {
+  installTemplate: jest.fn()
+}
+TemplateHandler.init.mockResolvedValue(mockTemplateHandlerInstance)
```

Comment:
@roribio , you mocked everything, but you don't check anything. I expected to see something like `expect(mockTemplateHandlerInstance.installTemplate).toBeCalledWith('org-id', 'project-id')` after https://github.com/adobe/aio-cli-plugin-app-templates/blob/DEVX-2298/test/commands/templates/install.test.js#L142, for example.

Reply:
Done :white_check_mark: